redis.conf

```diff
@@ -1426,15 +1450,12 @@ auto-aof-rewrite-min-size 64mb
 # will be found.
 aof-load-truncated yes
 
-# When rewriting the AOF file, Redis is able to use an RDB preamble in the
-# AOF file for faster rewrites and recoveries. When this option is turned
-# on the rewritten AOF file is composed of two different stanzas:
-#
-#   [RDB file][AOF tail]
-#
-# When loading, Redis recognizes that the AOF file starts with the "REDIS"
-# string and loads the prefixed RDB file, then continues loading the AOF
-# tail.
+# The base AOF rewrite encodeing type.
+# 
+# When rewriting the AOF file, Redis will generate a new incremental file and a 
```

Comment:
```suggestion
# When creating the initial AOF file (or rewriting it), Redis will generate a new incremental file and a 
```

Reply:
When we initialize (for example, when redis is started), we will open INCR AOF instead of creating a new BASE, so I think the original comments is fine.

Reply:
we're gonna change this soon (will always create a base file even in that case), see #9794 
but maybe we can find a way to say that without mentioning neither "initial", nor "rewrite" which is also not in line with the above mentioned plan.

I see this section is below all the text that defines the folder and base / incremental.
so maybe we can just refer to the "base"?

Reply:
OK